Android/SortableTableView/source/SortableTableView/TableView.cs

```diff
@@ -8,12 +8,6 @@ partial class TableView
     {
         private WeakReference dataClickListener;
 
-        public TableDataAdapter DataAdapter
-        {
-            get { return TableDataAdapter; }
-            set { SetDataAdapter(value); }
-        }
-
```

Comment:
This was added by the native java code, so we can chuck it out. basically, this resulted in duplicate members.